tests/integration/slurm_ops/test_manager.py

```diff
@@ -26,9 +26,6 @@ def test_install(slurmctld: SlurmctldManager, etc_path: Path) -> None:
     assert key == slurmctld.munge.key.get()
 
 
-"sz+VRDLFlr3o"
```

Comment:
Oh, this is the prefix of a mungekey that I was using for testing purposes (for the newline thing that we were talking about), not a password or something like that 😅

Reply:
Ah, I thought it was just like your computer fell asleep and you were just like "come back to me {PO+v[osndfkfjdgafdjbasd;fdbk;sadjbfkdsjg" :rofl: